### DISCUSSION TITLE
hi

### DISCUSSION THREAD

**Discussion by Serhiicrypto** [NillionNetwork/discussions#91] [General]

**Comment by oceans404** [comment]
Hey there, this Github Discussions forum is dedicated to developer discussions, where builders can share ideas, ask questions, showcase their projects, and stay updated on development-related topics. 

For any issues or concerns unrelated to these developer discussions, please refer to the appropriate platform. If your concern is related to Discord, please reach out to the moderators there for assistance.

**Comment by Serhiicrypto** [comment]
Hello.
Sorry, I deleted everything.
